test/support/organisation_helper.rb

```diff
@@ -359,4 +359,23 @@ def organisation_with_foi
       }
     }.with_indifferent_access
   end
+
+  def organisation_with_corporate_information
+    {
+      title: "Attorney General's Office",
+      details: {
+        brand: "attorney-generals-office",
+        ordered_corporate_information_pages: [
+          {
+            title: "Corporate Information page",
+            href: "/corporate-info"
+          },
+          {
+            title: "Jobs",
+            href: "/jobs"
```

Comment:
Do we need to add a few more links here to more thoroughly test the logic for separating job links?